shipyard_blueprints/portable/shipyard_portable/errors.py

```diff
@@ -0,0 +1,15 @@
+from shipyard_templates import ExitCodeException, Etl
+
```

Comment:
rename folder to exceptions to match the newer format and move common exceptions to abstract class

Reply:
in this case looks like both exceptions can be moved to the abstract class